src/deriver/__main__.py

```
@@ -1,11 +1,54 @@
 import asyncio
+import logging
+import os
 
 import uvloop
 
+from src.config import settings
+
 from .queue_manager import main
 
+
+def setup_logging():
```

Ruff (0.12.2)

Missing return type annotation for public function `setup_logging`

Add return type annotation: `None`

(ANN201)